lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -115,12 +115,32 @@
                     return DateUtils.localize(context);
                 },
 
+                getCertificatePriceString: function(run) {
+                    var verifiedSeat, currency;
+                    if ('seats' in run && run.seats.length) {
+                        // eslint-disable-next-line consistent-return
```

Comment:
I am using filter and not returning all seat types (which means the function doesn't always return.)  
Not sure, maybe i can return nothing to make the filter ignore it or something like that